langwrappers/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+    kotlin("jvm")
+    id("org.jetbrains.intellij")
+}
+
+repositories {
+    mavenCentral()
+    // Add any other repositories you need
+}
+
+dependencies {
+    implementation(kotlin("stdlib"))
+
+    implementation(project(":core"))
+}
+
+intellij {
+    rootProject.properties["platformVersion"]?.let { version.set(it.toString()) }
+    plugins.set(listOf("java"))
+    downloadSources.set(true)
+}
+
+tasks.named("verifyPlugin") { enabled = false }
+tasks.named("runIde") { enabled = false }
+tasks.named("runPluginVerifier") { enabled = false }
+
+kotlin {
+    jvmToolchain(17)
+}
```

Comment:
Here also a duplication of jvm 17.It should be moved into a variable, e.g., `JVM_TOOLCHAIN_VERSION`.